### PR TITLE
iOS multi server docs

### PR DESCRIPTION
**Pull request by TomBrien**
First thoughts on places that will need updating to reflect changes from multi-server being implemented

### CODE REVIEW

**Comment by TomBrien** [comment]
Some `prettier` auto linting sneaked in before I noticed and disabled it

File: docs/integrations/url-handler.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ Home Assistant for iOS supports opening from other apps via URL.
 
 Query parameters are passed as a dictionary in the call.
 
+If multiple servers are connected to an iOS or mac app, you will be prompted to select a server when handling a `navigate`, `call_service`, or `fire_event` link.
```
Comment (zacwest) [inline docs/integrations/url-handler.md:11]: I made these non-navigate only work on the first server.

**Review by zacwest — COMMENTED** [review]
lgtm so far

**Comment by TomBrien** [comment]
@zacwest can you think of anything major missed?

**Review by zacwest — APPROVED** [review]
lgtm

**Pull request merged**